parser/src/lexer.rs

```diff
@@ -31,7 +31,9 @@ impl<I> StreamOnce for LocatedStream<I>
                 self.location.bump(ch);
                 // HACK: The layout algorithm expects `1` indexing for columns -
                 // this could be altered in the future though
-                self.location.column += Column::from(1);
+                if self.location.column == Column::from(0) {
+                    self.location.column = Column::from(1);
+                }
```

Comment:
It didn't actually have any effect as the column is only used in the lexer and its only compared with other columns which have been double incremented.